source/computer/command-line.rst

```diff
@@ -35,3 +35,66 @@
    
    # 退出终端
    $ exit
+
+Shell
+------
+
+命令行实际上指的是 Shell（壳），它是一个用于接收键盘输入的命令，然后把命令交给系统执行的程
+序。用户通过终端程序输入命令时，其实是在与 Shell 交互，与系统直接交互的是 Shell。因此，
+终端和 Shell 都是程序，前者用于输入命令，后者则是接收命令然后交给系统执行。
+
+几乎所有的 Linux 发行版都提供了一个叫 Bash 的 Shell，此外还有 Zsh、csh、ksh 等 Shell。
+熟悉 Linux 系统之后，推荐参考\ :doc:`《Zsh 及其配置实践经验》<best-practices/zsh>`\
+安装和使用 Zsh。
+
+.. dropdown:: :fa:`exclamation-circle,mr-1` Linux 内核
+   :container: + shadow
+   :title: bg-info text-white font-weight-bold 
+
+   有壳（Shell）必有核（kernel）。kernel 的常用翻译有核、内核、核心等。内核才是操作系
+   统本体，是真正负责调度程序、管理硬件的部分。日常所说的 Linux 其实指的是内核。
+   
+   我们无法直接使用内核，日常使用的 Linux 操作系统其实是一个包含内核和一批有用程序
+   （如 Shell、终端）的集合体，即 Linux 发行版。Linux 发行版本身只是一个套装，把内核
+   用不同的程序包装起来。不同发行版的差异体现了他们之间不同的理念。
+   
+   `www.kernel.org <https://www.kernel.org/>`__ 网站负责存储和发布内核。使用以下命令
+   可以查询当前使用的 kernel 版本号::
+
+       # 不同 Linux 发行版显示略有区别
+       $ uname -r
+       5.6.19-300.fc32.x86_64
```

Comment:
This paragraph seems useless to me.

Reply:
I think it is a good description of the Linux kernel and shell, although it does not matter if new babies know it. I think we can leave it in the dropdown box so that new babies can know why we have a program named "Shell".

Reply:
I still think this part is unnecessary. I don't need to understand any differences or relationship between kernel and shell.

Reply:
new babies。我觉得这段描述说得不错。其实还可以说明一下终端和shell的关系，和终端为什么叫终端

Reply:
> new babies。我觉得这段描述说得不错。其实还可以说明一下终端和shell的关系，和终端为什么叫终端

这里描述了一点： https://seismo-learn.org/seismology101/computer/command-line/#id3

Reply:
> I still think this part is unnecessary. I don't need to understand any differences or relationship between kernel and shell.

Let's keep it for some time. We may revise it in the future.